blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -463,8 +479,9 @@ export class WorkspaceSearch {
 
   /**
    * Populates block list with blocks that match the search text.
+   * @protected
    */
-  populateBlocks() {
+  populateBlocks_() {
```

Comment:
I feel like the name of the function `populateBlocks`  hints at how it sets the `blocks_` array and someone overriding this function would have access to the array since its been made protected as well.
So, if we were to change to have it pass around array, then I'd also want a rename. 
The pattern of passing around the blocks array sounds like it could be nice, but I'd rather not introduce these changes now and save it for a future version of the plugin.

Reply:
If we wait until later and someone has already overridden this, then it will break them because they won't be returning the correct value. Would rather decide earlier rather than later how we want to implement this.